src/Pages/suna/Login/Login.scss

```
@@ -0,0 +1,99 @@
+.background {
+  width: 100%;
+  height: 100vh;
+  padding-top: 32px;
+  background-color: #fafafa;
+  .login-page {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    max-width: 350px;
+    margin: 0 auto;
+    color: #262626;
+    background-color: #fff;
+    border: 1px solid #dbdbdb;
+    border-radius: 1px;
+    .logo {
+      display: flex;
+      justify-content: center;
+      width: 175px;
+      margin: 48px auto 10px;
+      font-size: 2.6em;
+      font-family: "Lobster", cursive;
+      font-weight: 400;
+      a {
+        color: inherit;
+        text-decoration: none;
+      }
+    }
+    .login-box {
+      display: flex;
+      flex-direction: column;
+      align-items: center;
+      max-width: 350px;
+      width: 100%;
+      margin-bottom: 10px;
+      .input-box {
+        margin: 32px 0 16px 0;
```

아래와 같이 작성해주실 수 있습니다.
```suggestion
        margin: 32px 0 16px;
```